mbuild/compound.py

```diff
@@ -1226,22 +1517,22 @@ def energy_minimization(self, steps=2500, algorithm='cg',
                protein models implemented in an off-lattice force field" (2001)
                J. Comput. Chem. 22, 1229-1242
         """
+
         openbabel = import_('openbabel')
 
         for particle in self.particles():
             try:
                 get_by_symbol(particle.name)
             except KeyError:
-                raise MBuildError("Element name {} not recognized. Cannot "
-                                  "perform minimization."
-                                  "".format(particle.name))
+                try:
+                    get_by_symbol(particle.name[1:2])
```

Comment:
This should be changed back. Looks like this piece of code was leftover from the reverse mapping PR